### ISSUE TITLE
Trouble stubbing Godot 4.1

### ISSUE THREAD

**Issue by soft-circles** [bitwes/Gut#505]
In Godot 4.1 I just have this code in a test:
```
extends GutTest

var entity_state_stub = load('res://scenes/entity/purple_man/purple_man.gd')

var whatever = double(entity_state_stub)
```

when I run it I get the following error:

`Invalid call. Nonexistent function 'get doubler' in base 'Nil'.`

And it points me to this line in `res:://addons/gut/test.gd`

<img width="920" alt="Screenshot 2023-07-14 at 12 10 00 PM" src="https://github.com/bitwes/Gut/assets/4955407/69675fd3-ec90-447c-baa7-f53cc488f30e">

In `gut/test.gd` on line `49`:
```
var gut = null
```

Seems to not be set. Is there a problem with my configuration?

**Comment by bitwes** [comment]
Is your code inside a test function or is it at the script level?  You
can't double at the script level.  You can double in before_all to get the
same effect though.

**Comment by soft-circles** [comment]
I put it inside a test function and now I'm getting this error:
```
Parser Error: The method "add_sibling()" overrides a method from native class "Node". This won't be called by the engine and may not work as expected. (Warning treated as error.)
```

**Comment by bitwes** [comment]
This sounds related to #482.  I've implemented a fix for it but it hasn't
been released yet.  If you use the download link on the godot_4 branch I
think it will fix it.  Take a look at the workaround instructions in #482
and let me know if that fixes it.

**Comment by bitwes** [comment]
Just released version [9.1.0](https://github.com/bitwes/Gut/releases/tag/v9.1.0) which has the fix for this.  It may not be in the asset library yet but will be soon.

**Comment by soft-circles** [comment]
9.1.0 seems to have fixed the issue. Thank you!!!
